xtp/src/libxtp/dftengine/dftengine.cc

```diff
@@ -402,21 +411,22 @@ bool DFTEngine::EvaluateActiveRegion(Orbitals& orb) {
 
   Index all_electrons = static_cast<Index>(
       std::round(FullDensityMatrix.cwiseProduct(overlap.Matrix()).sum()));
-  Index active_electrons = static_cast<Index>(std::round(
+  active_electrons_ = static_cast<Index>(std::round(
       InitialActiveDensityMatrix.cwiseProduct(overlap.Matrix()).sum()));
   Index inactive_electrons = static_cast<Index>(
       std::round(InactiveDensityMatrix.cwiseProduct(overlap.Matrix()).sum()));
 
   XTP_LOG(Log::error, *pLog_)
       << TimeStamp() << " Total electrons: " << all_electrons << std::flush;
   XTP_LOG(Log::error, *pLog_)
-      << TimeStamp() << " Active electrons: " << active_electrons << std::flush;
+      << TimeStamp() << " Active electrons: " << active_electrons_
+      << std::flush;
   XTP_LOG(Log::error, *pLog_)
       << TimeStamp() << " Inactive electrons: " << inactive_electrons
       << std::flush;
 
   // check for consistency
-  if ((active_electrons + inactive_electrons) != all_electrons) {
+  if ((active_electrons_ + inactive_electrons) != all_electrons) {
     XTP_LOG(Log::error, *pLog_) << TimeStamp()
```

Comment:
Maybe this should throw a runtime error instead of just a message.

Reply:
Done